### DISCUSSION TITLE
Issue wth persistence

### DISCUSSION THREAD

**Discussion by amazingakai** [ventoy/Ventoy#3062] [Q&A]
having some problem with ventoy, i installed my iso and persistence file in correct place and created the ventoy.json file inside the ventoy directory, when i boot into the flash drive and select my iso, i get two options

1: boot without persistence
2: boot with persistence

after selecting 2, when i boot into the live environment and do some changes and restart the system, my changes get reset, am i doing something wrong

i use systemctl poweroff cmd to turn it off and then turn it back on normally.

**Comment by guillermodotn** [comment]
Which distro are you using?

**Reply by amazingakai** [reply]
This one: https://archcraft.io/

**Reply by guillermodotn** [reply]
Okay, and what about the persistence backend you are using? Did you generate it yourself?

**Reply by amazingakai** [reply]
Yes I generated it myself using this script: https://ventoy.net/download/CreatePersistentImg.sh

Generated it on Arch Linux.
